charts/values.yaml

```diff
@@ -8,6 +8,8 @@ openebs-crds:
 localpv-provisioner:
   rbac:
     create: true
+  analytics:
+    enabled: true
```

Comment:
What is this change doing, was it not enabled?

Reply:
Removed

Reply:
I mean, we can keep, was just questioning if this was a change because if it was then would need more documentation.